src/components/tasks/TaskDetails.tsx

```diff
@@ -348,6 +349,9 @@ function TaskDetails(props: Props, context) {
         <TaskCommandList task={task} />
       </Paper>
       <div className={classes.gap} />
+      <Paper elevation={2}>
```

Comment:
same as above 👌

Reply:
This was also fixed by https://github.com/cirruslabs/cirrus-ci-web/pull/358/commits/ae109a2f9038c94569ee35bd75f444f16b943718.

Reply:
I see that it's still after `TaskList` and after `TaskCommandsList` respectively 🤔

Reply:
Oh, I've thought we've sorted out the ordering issue as you've resolved the conversation above.

IMO navigating to the list of hooks in a long task list is as easy as pressing the `End` key. However, even without considering hooks, how would a user find the specific task in that huge list anyway?

What do you think about using the [`Tabs`](https://material-ui.com/components/tabs/) on a `<Paper>` that's currently displays tasks instead? The `Tasks` tab would be always shown, but `Hooks` tab will appear if there are any hooks.

Reply:
Implemented `Tabs` in https://github.com/cirruslabs/cirrus-ci-web/pull/358/commits/0c8e4062e45862f7f847b374359bfad3f026193b:

<img width="1123" alt="Screenshot 2021-04-20 at 03 16 01" src="https://user-images.githubusercontent.com/85709/115318681-e6838280-a186-11eb-9d56-290eb324f2fb.png">

<img width="1123" alt="Screenshot 2021-04-20 at 03 16 05" src="https://user-images.githubusercontent.com/85709/115318686-e8e5dc80-a186-11eb-8500-b0cd8e7517f4.png">

Light theme looks somewhat contrasty, but there's also a header (not displayed) that has a similar color, so if we're going to fix this we'll should probably fix both.